Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.4
```

Comment:
having some issues getting ldr to build on 3.6 right now, i'll play around and update the PR if i can get it to build


it's not finding cgo.a, but it's there...
```
/build # godep go build -a -installsuffix cgo -o ldr
godep: [WARNING]: godep should only be used inside a valid go package directory and
godep: [WARNING]: may not function correctly. You are probably outside of your $GOPATH.
godep: [WARNING]:	Current Directory: /build
godep: [WARNING]:	$GOPATH: 
godep: WARNING: Godep workspaces (./Godeps/_workspace) are deprecated and support for them will be removed when go1.8 is released.
godep: WARNING: Go version (go1.8) & $GO15VENDOREXPERIMENT= wants to enable the vendor experiment, but disabling because a Godep workspace (Godeps/_workspace) exists
# _/build
warning: unable to find runtime/cgo.a
/build # find / -name cgo.a
/usr/lib/go/pkg/linux_amd64/runtime/cgo.a
```